framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -1153,7 +1169,7 @@ public boolean isWithinInteger() {
      * @param bigTo the upper bound of the BigInteger range
      * @return a range with Long type bounds converted from the BigInteger range
      */
-    private static Range bigRangeToLongRange(BigInteger bigFrom, BigInteger bigTo) {
+    private static Range create(BigInteger bigFrom, BigInteger bigTo) {
```

Comment:
Move this up to the other `create` methods?

Reply:
Agreed. 
I edited it in place to make the review comparison easier to tell this was being renamed.

Reply:
Yes, that's a good strategy. In these cases, maybe add a TODO comment that you plan to move the method after the review.

Reply:
These two new create methods should be moved up in the file to the other create methods, as discussed above.

Reply:
See https://github.com/typetools/checker-framework/pull/2924#issuecomment-560803366